--- scap-security-guide-0.1.21/RHEL/6/input/guide.xslt.orig	2015-03-09 07:11:04.999165846 -0400
+++ scap-security-guide-0.1.21/RHEL/6/input/guide.xslt	2015-03-09 07:19:42.359027678 -0400
@@ -8,7 +8,8 @@
       <xsl:copy-of select="@*|node()" />
 
        <!-- adding profiles here -->
-		<xsl:apply-templates select="document('profiles/test.xml')" />
+		<!-- Don't include the 'test' profile per RH BZ#1199946 -->
+		<!-- <xsl:apply-templates select="document('profiles/test.xml')" /> -->
 		<xsl:apply-templates select="document('profiles/CS2.xml')" />
 		<xsl:apply-templates select="document('profiles/common.xml')" />
 		<!-- <xsl:apply-templates select="document('profiles/desktop.xml')" /> -->
--- scap-security-guide-0.1.21/RHEL/7/input/guide.xslt.orig	2015-03-09 07:11:57.924186953 -0400
+++ scap-security-guide-0.1.21/RHEL/7/input/guide.xslt	2015-03-09 07:18:50.334061326 -0400
@@ -8,7 +8,8 @@
       <xsl:copy-of select="@*|node()" />
 
        <!-- adding profiles here -->
-		<xsl:apply-templates select="document('profiles/test.xml')" />
+		<!-- Don't include the 'test' profile per RH BZ#1199946-->
+		<!-- <xsl:apply-templates select="document('profiles/test.xml')" /> -->
 		<xsl:apply-templates select="document('profiles/rht-ccp.xml')" />
 		<xsl:apply-templates select="document('profiles/common.xml')" />
 		<xsl:apply-templates select="document('profiles/stig-rhel7-server-upstream.xml')" />
